src/pcapi/routes/adage/v1/serialization/prebooking.py

```diff
@@ -0,0 +1,58 @@
+from datetime import datetime
+import enum
+from typing import Optional
+
+from pydantic import BaseModel
+
+from pcapi.routes.native.v1.serialization.common_models import Coordinates
+from pcapi.routes.native.v1.serialization.offers import OfferCategoryResponse
+from pcapi.routes.native.v1.serialization.offers import OfferImageResponse
+from pcapi.serialization.utils import to_camel
+
+
+class PreBookingStatuses(enum.Enum):
+    PENDING = "PENDING"
+    CONFIRMED = "CONFIRMED"
+    REFUSED = "REFUSED"
+    USED = "USED"
+
+
+class GetPreBookingsRequest(BaseModel):
+    schoolId: str
+    yearId: str
+    redactorEmail: Optional[str]
+    status: Optional[PreBookingStatuses]
+
+
+class PreBookingResponse(BaseModel):
+    address: str
+    beginningDatetime: datetime
+    cancellationDate: Optional[datetime]
+    category: OfferCategoryResponse
+    city: str
+    confirmationDate: Optional[datetime]
+    confirmationLimitDate: Optional[datetime]
+    coordinates: Coordinates
+    expirationDate: Optional[datetime]
+    id: int
+    image: Optional[OfferImageResponse]
+    isDigital: bool
+    name: str
+    name: str
+    postalCode: str
+    publicName: Optional[str]
```

Comment:
```suggestion
    redactorName: Optional[str]
```
Pourquoi le mettre en `Optional` ?

Reply:
C'était le publicName de la venue, j'ai enlevé : charge à nous de le mettre à la place du venueName quant il est là ?
C'est quoi redactorName pour toi ? Tu veux que je rajoute ?

Reply:
C'était le nom/prénom du rédacteur de projet
S'ils en on besoin ils nous le diront

Reply:
Je me dis que c'est mieux de leur envoyer le min sur le rédacteur vu qu'il a une existence du côté adage aussi
donc uniquement l'identifiant commun à savoir le mail
On verra...